dominos/src/main/java/me/fontys/semester4/dominos/pizza/and/ingredient/PizzaAndIngredientRecord.java

```diff
@@ -0,0 +1,125 @@
+package me.fontys.semester4.dominos.pizza.and.ingredient;
+
+import com.opencsv.bean.CsvBindByPosition;
+
+public class PizzaAndIngredientRecord {
+
+    @CsvBindByPosition(position = 0)
+    private String categorie;
+    @CsvBindByPosition(position = 1)
+    private String subcategorie;
+    @CsvBindByPosition(position = 2)
+    private String productnaam;
+    @CsvBindByPosition(position = 3)
+    private String productomschrijving;
+    @CsvBindByPosition(position = 4)
+    private String prijs;
+    @CsvBindByPosition(position = 5)
+    private String bezorgtoeslag;
+    @CsvBindByPosition(position = 6)
+    private String spicy;
+    @CsvBindByPosition(position = 7)
+    private String vegetarisch;
+    @CsvBindByPosition(position = 8)
+    private String beschikbaar;
+    @CsvBindByPosition(position = 9)
+    private String aantalkeer_ingredient;
+    @CsvBindByPosition(position = 10)
+    private String ingredientnaam;
+    @CsvBindByPosition(position = 11)
+    private String pizzasaus_standaard;
+
+    public PizzaAndIngredientRecord(){}
+
+    public PizzaAndIngredientRecord(String categorie,
+                                    String subcategorie,
```

Comment:
See earlier comment about a new line for every parameter

Reply:
Push inc.